ChangeLog

```diff
@@ -5,8 +5,7 @@ astroid's ChangeLog
 
 What's New in astroid 2.5.0?
 ============================
-Release Date: TBA
-
+Release Date: 2020-12-27
```

Comment:
This is already obsolete. My suggestion is to simply backport the code changes but keep the `__pkginfo__` and `ChangeLog` for a separate change in the day of the release.

Reply:
@PCManticore i will move back the Release Date in the ChangeLog. However i didn't change the `__pkginfo__`.
By the way, have you an idea concerning the release date? There are many people waiting for a release that support python 3.9.

Reply:
Thanks. Not right now. I am still going through the latest updates, mails and issues and don't have a clear picture on what's needed for Python 3.9 compatibility. I did notice some Python 3.9 issues specific to `pylint` though.